instrumentation-core/spring/README.md

```diff
@@ -2,11 +2,344 @@
+        restTemplate.exchange(url, HttpMethod.GET, entity, String.class);
+
+    return response.getBody();
+  }
+}
+```
+### SecondService
+
+1. Ensure OpenTelemetry dependencies are included
+2. Ensure an OpenTelemetry Tracer is configured
+3. Ensure a Spring Boot main class was created by the Spring initializer
+
+  
+```java
+import java.io.IOException;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SecondServiceApplication {
+
+  public static void main(String[] args) throws IOException {
+    SpringApplication.run(SecondServiceApplication.class, args);
+  }
+}
+```
+
+4. Create a RestController for SecondService
+5. Start a span to wrap the SecondServiceController
+
+```java
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+
+@RestController
+@RequestMapping(value = "/time")
+public class SecondServiceController {
+  @Autowired
+  private Tracer tracer;
+
+  @GetMapping
+  public String callSecondTracedMethod() {
+    Span span = tracer.spanBuilder("time").startSpan();
+    span.addEvent("SecondServiceController Entered");
+    span.setAttribute("what.am.i", "Tu es une legume");
+
+    try (Scope scope = tracer.withSpan(span)) {
+      return "It's time to get a watch";
+    } finally {
+      span.end();
+    }
+  }
+}
+```
+
+### Run FirstService and SecondService
+
+***To view your distributed traces ensure either LogExporter or Jaeger is configured in the OtelConfig.java file*** 
+
+To view traces on the Jaeger UI, deploy a Jaeger Exporter on localhost by running the command in terminal:
+
+`docker run --rm -it --network=host jaegertracing/all-in-one` 
+
+After running Jaeger locally, navigate to the url below. Make sure to refresh the UI to view the exported traces from the two web services:
+
+`http://localhost:16686`
+ 
+Run FirstService and SecondService from command line or using an IDE. The end point of interest for FirstService is `http://localhost:8080/message` and  `http://localhost:8081/time` for SecondService. Entering `localhost:8080/time` in a browser should call FirstService and then SecondService, creating a trace. To send a sample request enter the following in a browser of your choice:
+
+`http://localhost:8080/time`
+
+***Note: The default port for the Apache Tomcat is 8080. On localhost both FirstService and SecondService services will attempt to run on this port raising an error. To avoid this add `server.port=8081` to the resources/application.properties file. Ensure the port specified corresponds to port referenced by FirstServiceController.SS_URL. ***
```

Comment:
```suggestion
***Note: The default port for the Apache Tomcat is 8080. On localhost both FirstService and SecondService services will attempt to run on this port raising an error. To avoid this add `server.port=8081` to the resources/application.properties file. Ensure the port specified corresponds to port referenced by FirstServiceController.SECOND_SERVICE_URL. ***
```